### PR TITLE
Version 4.2.3

### PR DESCRIPTION
**Pull request by eatyourgreens**
A small fix for error handling in `auth.signIn` when either the login or password are incorrect.

### CODE REVIEW

**Comment by eatyourgreens** [comment]
I'm going to go ahead and `npm publish` this branch.

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "panoptes-client",
-  "version": "4.2.2",
+  "version": "4.2.3",
```
Comment (goplayoutside3) [inline package-lock.json:3]: Does the `packages` object here also need a version bump like in [176](https://github.com/zooniverse/panoptes-javascript-client/pull/176/files)?

Comment (eatyourgreens) [reply package-lock.json:3]: oh, good point. Let me check.

**Review by goplayoutside3 — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (eatyourgreens) [reply package-lock.json:3]: Should be fixed now.

**Pull request merged**